config/mocha.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  plugins: [
+    'mocha',
+  ],
+
+  env: {
+    'mocha': true,
+  },
+
+  rules: {
+    'mocha/handle-done-callback': 'error',
+    'mocha/max-top-level-suites': [ 'error', { limit: 1 } ],
+    'mocha/no-async-describe': 'error',
+    'mocha/no-exclusive-tests': 'error',
+    'mocha/no-global-tests': 'error',
+    'mocha/no-hooks': 'error',
+    'mocha/no-hooks-for-single-case': 'error',
+    'mocha/no-identical-title': 'error',
+    'mocha/no-mocha-arrows': 'error',
+    'mocha/no-nested-tests': 'error',
+    'mocha/no-pending-tests': 'error',
+    'mocha/no-return-and-callback': 'error',
+    'mocha/no-return-from-async': 'error',
+    'mocha/no-setup-in-describe': 'error',
+    'mocha/no-sibling-hooks': 'error',
+    'mocha/no-skipped-tests': 'error',
+    'mocha/no-synchronous-tests': 'off',
+    'mocha/no-top-level-hooks': 'error',
+    'mocha/prefer-arrow-callback': 'off',
+    'mocha/valid-suite-description': 'error',
```

Comment:
>By default, the regular expression is not configured and would be required if rule is enabled

My interpretation is that this does nothing unless a regex is given.

I'd suggest maybe adding a simple regex just to ensure the description isn't empty? e.g. `^\w` 🤷‍♂

Reply:
Oh that's a good idea! I'll update this.

Reply:
I've added simple regex for the test descriptions too